EasyPost/Models/API/Order.cs

```diff
@@ -116,12 +115,12 @@ public async Task GetRates()
         /// <returns>Lowest EasyPost.Rate object instance.</returns>
         public Rate LowestRate(List<string>? includeCarriers = null, List<string>? includeServices = null, List<string>? excludeCarriers = null, List<string>? excludeServices = null)
         {
-            if (rates == null)
+            if (Rates == null)
             {
                 throw new Exception("Rates not populated.  Call GetRates() first.");
             }
 
-            return Rates.GetLowestObjectRate(rates, includeCarriers, includeServices, excludeCarriers, excludeServices);
+            return Calculation.Rates.GetLowestObjectRate(Rates, includeCarriers, includeServices, excludeCarriers, excludeServices);
```

Comment:
How did this not fail earlier if we were referencing the wrong thing?

Reply:
It was always pointing to the right place. The namespace of the function just wasn't fully explicit (was implicit by import instead).